findbugs/etc/messages.xml

```diff
@@ -1620,6 +1620,20 @@ factory pattern to create these objects.</p>
       ]]>
       </Details>
   </Detector>
+  <Detector class="edu.umd.cs.findbugs.detect.MonitoringValueBasedClass">
+    <Details>
+      <![CDATA[
+        <p>TODO</p>
+      ]]>
+      </Details>
+  </Detector>
+  <Detector class="edu.umd.cs.findbugs.detect.IdentityHashCodeOnValueBasedClass">
+    <Details>
+      <![CDATA[
+        <p>TODO</p>
```

Comment:
You should fill the detector information here and (more important) create information sections about separate bug patterns (`<BugPattern type="VBC_MO_WAIT">`, etc.) and the information about `VBC` group (`<BugCode abbrev="VBC">Value-based classes</BugCode>`). Put them into the separate groups.